### ISSUE TITLE
Bug: Core Input Group a11y screen reader issues

### ISSUE THREAD

**Issue by kavya-b** [SAP/fundamental-ngx#4229]
#### Is this a bug, enhancement, or feature request?
bug
#### Briefly describe your proposal.
Screen Reader(like Voiceover on Mac) does not read out the following:
-  form messages in the states example
<img width="428" alt="Screenshot 2021-01-04 at 4 40 50 PM" src="https://user-images.githubusercontent.com/53509521/103530079-d46a2680-4eac-11eb-81de-ef40a1cb2a75.png">

- the presence of the clear text icon (X) through an aria-label or tooltip.
<img width="1132" alt="Screenshot 2021-01-04 at 4 41 53 PM" src="https://user-images.githubusercontent.com/53509521/103530096-d9c77100-4eac-11eb-8070-b0aaf828b6c9.png">

#### Which versions of Angular and Fundamental Library for Angular are affected? (If this is a feature request, use current version.)
latest v0.26.0-rc.28
#### If this is a bug, please provide steps for reproducing it.
1. Go to https://fundamental-ngx.netlify.app/#/core/inputGroup
2. Turn on Voiceover in Mac, and navigate through the examples.
3. Go to example `Input Group with States.` - the form messages are not read out.
4. Go to example ` Input Group Search` and write in the 2nd input group such that a X icon appears next to the search icon. The existence of the clear icon(X) is not notified by screen reader.
